migrations/versions/88040ddc0b08_move_data_and_times.py

```diff
@@ -0,0 +1,50 @@
+"""Move data and times into summary table
+
+Revision ID: 88040ddc0b08
+Revises: e06cede49f65
+Create Date: 2022-03-30 16:58:17.412915
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import text
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "88040ddc0b08"
+down_revision = "e06cede49f65"
+branch_labels = None
+depends_on = None
+
+
+def set_data_and_times():
+    connection = op.get_bind()
+    connection.execute(
+        text(
+            """UPDATE summary
+           SET data = ARRAY(SELECT result FROM data WHERE summary_id = summary.id ORDER BY iteration),
+           times = ARRAY(SELECT result FROM "time" WHERE summary_id = summary.id ORDER BY iteration);
```

Comment:
I don't think we need to change this, because we aren't using this data in this way: but technically what we would want is to construct the array based on the actual value of iteration so if one was missing (e.g. we have times for iterations 1, 2, 4, 5 but 3 has no time we would want `[x, x, NULL, x, x]`.

But again, let's leave this be so that it's simpler and if we get to the point where some iterations fail, we can deal with that later. And as far as I know we don't have any examples of that at all now (and even if we did we don't look at those in that way regardless)

Reply:
I will look into this before merging PR.

Reply:
I just ran a check to see if we have any occurences of this at all, and we currently do not. Both of the queries below returned no rows:

```
select n, max_iter, summary_id from (select count(*) n, max(iteration) max_iter, summary_id 
from public.time
group by summary_id) as tbl
where n != max_iter;

select n, max_iter, summary_id from (select count(*) n, max(iteration) max_iter, summary_id 
from public.data
group by summary_id) as tbl
where n != max_iter;
```

So I don't think we need to worry about this at all as part of the migration.

Reply:
I did the same!  and did not find anything.
```sql
select summary_id, count(1), max(iteration)
from data
group by summary_id
having by count(1) != max(iteration)

select summary_id, count(1), max(iteration)
from time
group by summary_id
having by count(1) != max(iteration)
```